circle.yml

```diff
@@ -2,7 +2,7 @@ machine:
   environment:
     DOCKER_IMAGE_NAME: prom/node-exporter
     QUAY_IMAGE_NAME: quay.io/prometheus/node-exporter
-    DOCKER_TEST_IMAGE_NAME: quay.io/prometheus/golang-builder:1.6-base
+    DOCKER_TEST_IMAGE_NAME: quay.io/prometheus/golang-builder:1.7-base
```

Comment:
I just want to make sure that we get builds from CircleCI to use 1.7.4 for the next release.